CHANGES.md

```diff
@@ -1,3 +1,33 @@
+1.7.2 / 2019-09-XX
+==================
+
+Bug fixes:
+
+* DriverInterface#getValue() can return null
+* Modify response[Contains|NotContains] to use strpos vs preg_match
```

Comment:
should be removed IMO. This is a change of an internal implementation, without any user-facing impact.

Reply:
I wanted to keep it because I remember discussions, where person was using API in incorrect way (knowing that `preg_match` is used internally and specified regular expressions in the input) started to complain about change to `stripos`, because his tests started to fail for no reason.

If that is no longer the case, then we can safely remove this line.